ui/app/components/config-pki-ca.js

```diff
@@ -121,7 +128,14 @@ export default Component.extend({
       },
     ];
     if (caChain) {
-      const caChainFile = new File([caChain], { type: 'text/plain' });
+      let caChainFile;
+
+      // this is a sad workaround because ie11 doesn't support Files.
+      try {
+        caChainFile = new File([caChain], { type: 'text/plain' });
+      } catch (err) {
+        caChainFile = new Blob([caChain], { type: 'text/plain' });
+      }
```

Comment:
Is there any reason to not use `Blob` everywhere for these two? I don't think we're doing anything `File`-specific here since they're just download links.

Reply:
oh, good point. i think it should be safe, yes. thanks!